Cargo.toml

```diff
@@ -3,17 +3,18 @@ name = "esp-println"
 version = "0.1.0"
 edition = "2021"
 
-[dependencies]
-
 [features]
-esp32 = []
+default = ["uart"]
+
+# You must enable exactly 1 of the below features to support the correct chip:
+esp32   = []
 esp32c3 = []
 esp32s2 = []
 esp32s3 = []
 esp8266 = []
 
-uart = []
-jtag_serial = [] # C3 + S3
-rtt = []
-
-default = [ "uart" ]
+# You must enable exactly 1 of the below features to enable to intended
+# communication method (note that "uart" is enabled by default):
+uart        = []
+jtag_serial = [] # C3 + S3 only!
```

Comment:
Maybe we should check this in the build script too?

Reply:
Sorry what do you mean? It already checks that exactly one of `uart`, `jtag_serial`, or `rtt` are enabled.

Reply:
Sorry! I meant that jtag_serial is only allowed on the C3 or S3.

Reply:
All good, I figured it out eventually 😂 I've updated the script.